test/trento/application/usecases/hosts_test.exs

```diff
@@ -25,4 +25,16 @@ defmodule Trento.HostsTest do
       assert 6 = Hosts.get_all_sles_subscriptions()
     end
   end
+
+  describe "Hosts listing" do
+    test "should filter unregistered hosts" do
+      %{id: host_id} = insert(:host)
```

Comment:
maybe `insert_list(10, :host)`

Reply:
I used an Enum.map so that the resulting `registered_hosts` was sorted alphabetically (otherwise the assertion below fails due to different ordering as `get_all_hosts` uses `sort`)